arangod/Cluster/FollowerInfo.h

```diff
@@ -48,12 +48,12 @@ class Slice;
 
 class FollowerInfo {
   // This is the list of real local followers
-  std::shared_ptr<std::vector<ServerID> const> _followers;
+  std::shared_ptr<std::vector<ServerID>> _followers;
```

Comment:
I guess this is only to enable sorting and comparing them in maintainer mode. I think it'd be better to not generally sacrifice const correctness for a maintainer-only check. Additionally, this currently changes the behaviour in maintainer mode compared to non-maintainer mode, because now those arrays will sorted in one case, but not in the other.

I'd recommend to copy them instead and sort the copies for the symmetric difference check.

Reply:
Outdated